dataflow/run_template/main.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+#
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+
+"""Script to run a Dataflow template."""
+
+
+def run(project, job, template, parameters=None):
+    """Runs a Dataflow template.
+
+    Args:
+        project (str): Google Cloud project ID to run on.
+        job (str): Unique Dataflow job name.
+        template (str): Google Cloud Storage path to Dataflow template.
+        parameters (dict): Dictionary of parameters for the specified template.
+    Returns:
+        The response from the Dataflow service after running the template.
+    """
+    parameters = parameters or {}
+
+    # [START un_template]
```

Comment:
looks like a typo here, should be dataflow_run_template

Reply:
Good catch, fixed